server/nightmare/script.js

```diff
@@ -1,25 +1,32 @@
-var Nightmare = require('nightmare');
-var Promise = require('bluebird');
+//var Nightmare = require('nightmare');
+//var Promise = require('bluebird');
+var Screenshot = require('url-to-screenshot');
+var fs = require('fs');
+var cloudinary = require('cloudinary');
 
-//var width = require('');
+var width = 800 //require('').width;
+var height = 600//require('').height;
 //var height = require('');
 
-var takeScreenshot = function(url) {
+var takeScreenshot = function(url, cb) {
     var randomString = Math.random().toString(36).substring(7);
-
-    new Nightmare()
-    //.viewport(900, 900)
-    .goto(url)
-    .screenshot('./screenshotsImages/' + randomString + '.png')
-    .title(function(str) {
-      console.log(str);
-    })
-    .run(function(err, nightmare) {
-      console.log(nightmare);
+    var fileName = "./" + randomString + ".png";
+    Screenshot(url)
+    .width(width)
+    .height(height)
+    .capture(function(err, img) {
+      if(err) throw err;
+      //fs.writeFile(fileName, img, function(err) {
+        //if(err) throw err
+        //console.log("img saved!!");
+      //})
+      cloudinary.uploader.upload(randomString + ".png", function(result) {
```

Comment:
I think you need to use `fs.createReadStream`. See here: https://devcenter.heroku.com/articles/cloudinary#using-with-node-js-upload. It's down a little bit.